plugins/SitesManager/SitesManager.php

```diff
@@ -88,6 +93,12 @@ public function redirectDashboardToWelcomePage(&$module, &$action)
     }
 
     public static function hasTrackedAnyTraffic($siteId)
+    {
```

Comment:
This method is also used in a task, does it need `shouldPerormEmptySiteCheck ()` there?